src/initialization/InitDistribution.cpp

```diff
@@ -357,6 +367,9 @@ namespace impactx
             amrex::ParticleReal * const AMREX_RESTRICT px_ptr = px.data();
             amrex::ParticleReal * const AMREX_RESTRICT py_ptr = py.data();
             amrex::ParticleReal * const AMREX_RESTRICT pt_ptr = pt.data();
+            amrex::ParticleReal * const AMREX_RESTRICT sx_ptr = sx.data();
+            amrex::ParticleReal * const AMREX_RESTRICT sy_ptr = sy.data();
+            amrex::ParticleReal * const AMREX_RESTRICT sz_ptr = sz.data();
```

Comment:
Could make this a `nullptr` on `!has_spin`, just in case:
https://en.cppreference.com/w/cpp/container/vector/data.html